build/yamls/antrea-aks.yml

```diff
@@ -4124,6 +4127,11 @@ data:
     # 3. The Node IP
     #transportInterface:
 
+    # The names of the interfaces on Node which are connected to external devices to the antrea clusters.
+    # The interface can be used to forward multicast traffic from and to the cluster
+    # if the Multicast config of Antrea agent is set to true.
```

Comment:
, when the Multicast feature is enabled.

Reply:
update